Fix build issue on MIPS64

On MIPS64, there is a syntax problem in the preprocessor
conditions. Fix that to avoid a build failure.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/mpi/longlong.h
===================================================================
--- a/mpi/longlong.h
+++ b/mpi/longlong.h
@@ -752,7 +752,7 @@
     (w1) = _r >> 64;                                                    \
     (w0) = (UDItype) _r;                                                \
   } while (0)
-# elif if __GNUC__ > 2 || __GNUC_MINOR__ >= 7
+# elif __GNUC__ > 2 || __GNUC_MINOR__ >= 7
 #  define umul_ppmm(w1, w0, u, v)                                       \
      __asm__ ("dmultu %2,%3"                                            \
               : "=l" ((UDItype)(w0)),                                   \
